icd/api/include/compiler_solution.h

```diff
@@ -93,6 +93,7 @@ struct GraphicsPipelineCreateInfo
     bool                                   freeWithCompiler;
     Util::MetroHash::Hash                  basePipelineHash;
     PipelineCreationFeedback               pipelineFeedback;
+    PipelineCreationFeedback stageFeedback[ShaderStage::ShaderStageGfxCount];
```

Comment:
This should probably be guarded by `LLPC_CLIENT_INTERFACE_MAJOR_VERSION`

Reply:
Done, but I do not understand what is the purpose of this. Is it used to prevent the stable version of ICD from including this new change to the release?

Reply:
Older ICD versions should be able to use newer LLPC. These guards allow the old ICDs to use old versions of the LLPC interface.

Reply:
done